assets/src/blocks/CarouselHeader/CarouselHeaderFrontend.js

```diff
@@ -35,9 +38,9 @@ export const CarouselHeaderFrontend = ({ slides, carousel_autoplay }) => {
       goToPrevSlide
     ];
 
-    // if (isRTL) {
-    //   swipeListeners.reverse();
-    // }
+    if (isRTL) {
+      swipeListeners.reverse();
```

Comment:
For readability it's significantly more straightforward if we don't create the array and instead check with a ternary in place.
```js
hammer.on('swipeleft', isRTL ? goToPrevSlide : goToNextSlide);
hammer.on('swiperight', isRTL ? goToNextSlide : goToPrevSlide);